Description: fix wrong variable name when setting host_id in ubuntu-cloud
 This is not needed upstream as it was inadvertently fixed when moving to
 a clone hook.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1197357
Forwarded: not-needed

Index: lxc-0.9.0/templates/lxc-ubuntu-cloud.in
===================================================================
--- lxc-0.9.0.orig/templates/lxc-ubuntu-cloud.in	2013-08-15 14:40:25.182793920 -0500
+++ lxc-0.9.0/templates/lxc-ubuntu-cloud.in	2013-08-15 14:40:43.402793735 -0500
@@ -368,7 +368,7 @@ mkdir -p @LOCALSTATEDIR@/lock/subsys/
 
         seed_d=$rootfs/var/lib/cloud/seed/nocloud-net
         rhostid=$(uuidgen | cut -c -8)
-        host_id=${hostid:-$rhostid}
+        host_id=${host_id:-$rhostid}
         mkdir -p $seed_d
 
         cat > "$seed_d/meta-data" <<EOF
